setup.py

```diff
@@ -93,7 +94,10 @@ def __init__(self):
         super(Protobuf, self).__init__()
         # TODO: allow user specify protobuf include_dirs libraries with flags
         # and environment variables
-        self.libraries = ['protobuf']
+        if os.getenv('CONDA_PREFIX'):
+            self.libraries = [os.path.join(os.getenv('CONDA_PREFIX'), "Library", "lib", "libprotobuf")]
```

Comment:
This is not portable. You should test if you're on Windows before doing this.

Reply:
it works in windows btw :).

Reply:
It works in Windows, but it breaks for other platforms. If you build in Linux conda environment, `self.libraries` should be `['protobuf']`, not `[os.path.join(os.getenv('CONDA_PREFIX'), "Library", "lib", "libprotobuf")]`.

Your `if` only checks for conda, but you should also check for platform

Reply:
Thank you very much! Limiting it only setting include dir and library for windows only now.